packages/plugin-core/src/features/TipOfTheDayWebview.ts

```diff
@@ -0,0 +1,182 @@
+import { VSCodeEvents } from "@dendronhq/common-all";
+import _md from "markdown-it";
+import * as vscode from "vscode";
+import {
+  DisplayLocation,
+  FeatureShowcaseUserResponse,
+  IFeatureShowcaseMessage,
+} from "../showcase/IFeatureShowcaseMessage";
+import { AnalyticsUtils } from "../utils/analytics";
+
+/**
+ * Side Panel webview that shows the tip of the day
+ * TODO: Add functionality
+ *  - let user rotate tips
+ *  - save which tip they're on. (right now it just shows the first one.)
```

Comment:
for mvp, can it show a random tip vs first one?

Reply:
Yeah - was planning on adding that logic in this revision.